StoreManagementService/src/PBJ.StoreManagementService.Api/Controllers/CommentController.cs

```diff
@@ -0,0 +1,73 @@
+﻿using AutoMapper;
+using FluentValidation;
+using Microsoft.AspNetCore.Mvc;
+using PBJ.StoreManagementService.Business.Services.Abstract;
+using PBJ.StoreManagementService.Models.Comment;
+
+namespace PBJ.StoreManagementService.Api.Controllers
+{
+    [ApiController]
+    [Route("api/comment")]
+    public class CommentController : ControllerBase
+    {
+        private readonly ICommentService _commentService;
+        private readonly IValidator<CommentRequestModel> _validator;
+
+        public CommentController(ICommentService commentService,
+            IValidator<CommentRequestModel> validator)
+        {
+            _commentService = commentService;
+            _validator = validator;
+        }
+
+        [HttpGet, Route("get/amount")]
+        public async Task<ActionResult> GetAmountAsync(int amount)
```

Comment:
complicated routes? could we simplify it?

![image](https://github.com/Genresal/PBJ/assets/38715090/2120cc15-52f2-4427-b38b-907736ee0cc7)